src/components/molecules/header/Header.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect } from "react";
+//TODO: IMPORT USESTATE
+//TODO: IMPORT LINK FROM ROUTER DOM
+
+//IMPORT COMPONENTS
+import Logo from "@atoms/logo/desktop/DesktopLogo";
+import Link from "@atoms/link/Link";
+import Menu from "@atoms/menu/Menu";
+
+//TODO: IMPORT STYLES
+import {
+  HeaderWrapper,
+  HeaderContainer,
+  HeaderLeftContainer,
+  HeaderContainerRight,
+  Line,
+} from "./Header.styles";
+
+// IMPORT ASSETS
+import { languages } from "@utils";
+
+const Header = () => {
+  // HOOKS
+  const [isMobile, setIsMobile] = useState(false);
+
+  useEffect(() => {
+    const handleResize = () => {
+      setIsMobile(window.innerWidth <= 700);
+    };
+
+    handleResize();
+    window.addEventListener("resize", handleResize);
+
+    return () => {
+      window.removeEventListener("resize", handleResize);
+    };
+  }, []);
+  // LIFECYCLES
+  // HANDLE FUNCTIONS
+  const handleTranslation = link => {
+    alert(link);
+  };
+  const handleLink = link => {
+    alert(link);
+  };
+
+  // DATA
+  const links = [
+    { label: "When", href: "when" },
+    { label: "Where", href: "where" },
+    { label: "How", href: "how" },
+  ];
+
+  // RENDER FUNCTIONS
+
+  // MAIN RENDER
+  return (
+    <HeaderWrapper>
+      <HeaderContainer>
+        <HeaderLeftContainer>
+          <Logo />
+          <nav>
+            {links.map(link => (
+              <Link label={link.label} onClick={() => handleLink(link.href)} />
+            ))}
+          </nav>
+        </HeaderLeftContainer>
+        {!isMobile && (
```

Comment:
Why only hide the language menu. Would it not be better to hide it all and put all the menu elements inside the future burger? Also in responsive when i click on the burger nothing happen. I don't see a menu that appear.

You are missing an important part of the functionality here.

Reply:
done